src/targets/maven.ts

```diff
@@ -213,30 +211,29 @@ export class MavenTarget extends BaseTarget {
    * @returns the gradle home path.
    */
   public getGradleHomeDir(): string {
-    if (process.env.GRADLE_USER_HOME) {
-      return process.env.GRADLE_USER_HOME;
-    }
-
-    return DEFAULT_GRADLE_USER_HOME;
+    return process.env.GRADLE_USER_HOME || DEFAULT_GRADLE_USER_HOME;
   }
 
   /**
    * Uploads the artifacts with the required files. This is a required step
    * to make a release, but this doesn't perform any releases; after upload,
-   * the flow must finish with `closeAndRelease`.
+   * the flow must finish with `closeAndReleaseRepository`.
    */
   public async upload(revision: string): Promise<void> {
-    this.logger.debug('Fetching artifact list...');
-    const artifacts = await this.getArtifactsForRevision(revision);
+    const artifacts = await this.getArtifactsForRevision(revision, {
+      includeNames:
+        this.config.includeNames === undefined
+          ? undefined
+          : stringToRegexp(this.config.includeNames),
```

Comment:
Do you really need this? I thought `getArtifactsForRevision` does this automatically? Or this should be handled in the config init section, not here.

Reply:
`getArtifactForRevision` expects `RegExp | undefined`, `includeNames` is `string | undefined`, and `stringToRegexp` expects `string`. So there must be a check for `undefined` somewhere.